test/Microsoft.TestPlatform.AcceptanceTests/BlameDataCollectorTests.cs

```diff
@@ -238,7 +237,7 @@ private void ValidateDump(int expectedDumpCount = 1)
                 // or WerFault in case we would start hanging during crash
                 // we don't want these to make cross-platform checks more difficult
                 // so we filter them out.
-                .Where(a => !a.Contains("WerFault") && !a.Contains("conhost"))
+                .Where(a => a.EndsWith(".dmp") && !a.Contains("WerFault") && !a.Contains("conhost"))
```

Comment:
Why we have new condition here?

Reply:
removed extra condition that was inserted by mistake :)

Reply:
EndsWith condition is still here.

Reply:
Little confused here. 
The original code was
`            var dumps = attachments
                .Where(a => a.EndsWith(".dmp"))
                // On Windows we might collect conhost which tells us nothing 
                // or WerFault in case we would start hanging during crash
                // we don't want these to make cross-platform checks more difficult
                // so we filter them out.
                .Where(a => !a.Contains("WerFault") && !a.Contains("conhost"))
                .Select(a => a.Trim()).ToList();`
               but now i've moved the condition (.dmp) into the where statement to not break the functionality.
               `            var dumps = attachments
                // On Windows we might collect conhost which tells us nothing
                // or WerFault in case we would start hanging during crash
                // we don't want these to make cross-platform checks more difficult
                // so we filter them out.
                .Where(a => a.EndsWith(".dmp") && !a.Contains("WerFault") && !a.Contains("conhost"))
                .Select(a => a.Trim()).ToList();`
                
So i can. remove the `a => a.EndsWith(".dmp")`, but i think that's incorrect as it was originally used a few lines up above the comment.

Reply:
Sorry, missed that part. Now it is ok.